packages/libs/react-ui/src/components/Tabs/Tabs.stories.tsx

```diff
@@ -27,45 +36,67 @@ const meta: Meta<
         type: { summary: 'number' },
       },
     },
+    initialTab: {
+      options: [
+        ...['-'],
+        ...Object.values(ExampleTabs),
+      ] as (keyof typeof ExampleTabs)[],
+      control: {
+        type: 'select',
+      },
+      description:
+        'The default selected page <em>before</em> any interaction.<br /><small>Changing value will not trigger story re-render.</small>',
+      table: {
+        defaultValue: { summary: 'undefined' },
+        type: { summary: 'string | undefined' },
+      },
+    },
+    currentTab: {
+      options: [
+        ...[undefined],
+        ...Object.values(ExampleTabs),
+      ] as (keyof typeof ExampleTabs)[],
+      control: {
+        type: 'select',
+      },
+      description:
+        'Current active tab. Used when component is controlled.<br /><small>Set to make component controlled.</small>',
+      table: {
+        defaultValue: { summary: 'undefined' },
+        type: { summary: 'string | undefined' },
+      },
+    },
   },
 };
 
-const ItemArray: string[] = [
-  'He-man',
-  'Skeletor',
-  'Orko',
-  'Teela-Na',
-  'Cringer',
-  'King Randor',
-];
-
 export default meta;
 type Story = StoryObj<ITabsContainerProps & { itemsCount: number }>;
 
 export const Primary: Story = {
   name: 'Tabs',
   args: {
-    itemsCount: 3,
-    defaultSelected: 'He-man',
+    itemsCount: 6,
+    initialTab: 'Skeletor',
+    currentTab: undefined,
   },
-  render: ({ itemsCount, defaultSelected }) => {
-    const items = ItemArray.slice(0, itemsCount);
+  render: ({ itemsCount, initialTab, currentTab }) => {
+    const tabs = ExampleTabs.slice(0, itemsCount);
 
     return (
       <>
-        <Tabs.Root defaultSelected={defaultSelected}>
-          {items.map((item) => {
+        <Tabs.Root initialTab={initialTab} currentTab={currentTab}>
+          {tabs.map((tab) => {
             return (
-              <Tabs.Tab key={item} value={item}>
-                {item}
+              <Tabs.Tab key={tab} id={tab}>
+                {tab}
               </Tabs.Tab>
```

Comment:
This is unrelated to this PR, but do you think we should wrap the tabs in a `Tabs.List`?

Reply:
Yeah I was thinking that. Personally I would opt to refactor the component a bit anyway and rethink its structure a bit. But not as part of this PR.

Reply:
Made a ticket to discuss later 👍